Makefile

```diff
@@ -14,15 +16,17 @@ endif
 all: manager
 
 # Run tests
-test: generate fmt vet manifests
-	go test ./... -coverprofile cover.out
+test: generate tidy fmt vet manifests
+	kind create cluster --name kms-issuer-test && \
+	USE_EXISTING_CLUSTER=$(USE_EXISTING_CLUSTER) go test ./... -coverprofile cover.out
```

Comment:
If `USE_EXISTING_CLUSTER` is `false`, are you still creating and deleting the kind clusters?

Reply:
yes, `USE_EXISTING_CLUSTER` is for `kubebuilder`'s test suite to run on existing cluster instead of instantiated `etcd` and `api-server`

Reply:
if it's set to `false` tests will fail

Reply:
I'll configure it so if the value of `USE_EXISTING_CLUSTER` is set to `true` `kind` cluster is started

Reply:
Yeah, I guess my question was around that if you set USE_EXISTING_CLUSTER to `false`, and then you run a `make test`, it will still create and delete a kind cluster.